frontend/src/app/app.component.css

```diff
@@ -0,0 +1,28 @@
+/* AppComponent's private CSS styles */
```

Comment:
are styles copied from the tutorial?
I think they might be refactored (class="informative-name" added and unused style statements removed)